mobius-migration/src/main/java/org/simple/mobius/migration/MobiusTestFixture.kt

```diff
@@ -38,7 +39,15 @@ class MobiusTestFixture<M: Any, E, F>(
       effectHandler: EffectHandler<F, E>,
       modelUpdateListener: ModelUpdateListener<M>,
       requiresLogging: Boolean = false
-  ) : this(events, defaultModel, initFunction, Update { model: M, event: E -> updateFunction(model, event) }, effectHandler, modelUpdateListener, requiresLogging)
+  ) : this(
+      events,
+      defaultModel,
+      Init<M, F> { initFunction?.invoke(it) ?: First.first(it) },
```

Comment:
I checked just now, and it seems like explicitly defining the type as `<M, F>` is unnecessary and we can just leave it as `Init {}`. Please check?

Reply:
Fixed!